docs/r1/overview.md

```diff
@@ -311,15 +311,15 @@ method.
 
 ### Can I customize which lines appear in a plot?
 
-Using the [custom scalars plugin](tensorboard/plugins/custom_scalar), you can
+Using the [custom scalars plugin](https://github.com/tensorflow/tensorboard/tree/master/tensorboard/plugins/custom_scalar), you can
```

Comment:
Could you use a relative path? i.e., `../../../tensorboard/plugins/custom_scalar`. Here and below

Reply:
Sure! I'll work on fixing the other two broken links you pointed out too!

Reply:
Thank you! For context, the reason that we like relative links is that
they enable editors to cross-link. For instance, in Vim, I can just
press `gf` (go-file) with my cursor on the relative link and it’ll open
in my editor.